src/frontend/src/btc/types/btc.ts

```diff
@@ -0,0 +1,20 @@
+import type { TransactionType } from '$lib/types/transaction';
+
+export type BtcTransactionStatus = 'confirmed' | 'pending';
+
+export interface BtcTransactionUi {
```

Comment:
Shouldn't we first implement the effective loading of the transactions before implementing this?